extension/src/experiments/studio.ts

```diff
@@ -1,12 +1,14 @@
 import { StudioLinkType } from './webview/contract'
 import { AvailableCommands, InternalCommands } from '../commands/internal'
 import { DeferredDisposable } from '../class/deferred'
+import { DEFAULT_STUDIO_URL } from '../setup/webview/contract'
 
 export class Studio extends DeferredDisposable {
   private readonly dvcRoot: string
   private readonly internalCommands: InternalCommands
 
-  private baseUrl: string | undefined = undefined
+  private url: string = DEFAULT_STUDIO_URL
+  private baseViewUrl: string | undefined = undefined
```

Comment:
[Q] Why change the name from `baseUrl`?

Reply:
I think `viewUrl` would be better. `url` should probably be `instanceUrl` or something like that

Reply:
I changed `baseUrl` to `baseViewUrl` to better differentiate between the two URLs, but I like your idea better :)

Reply:
Updated!